core/src/main/java/org/testcontainers/lifecycle/Startables.java

```diff
@@ -27,8 +30,8 @@ public Thread newThread(Runnable r) {
     /**
      * @see #deepStart(Stream)
      */
-    public CompletableFuture<Void> deepStart(Collection<? extends Startable> startables) {
-        return deepStart(startables.stream());
+    public CompletableFuture<Void> deepStart(Iterable<? extends Startable> startables) {
```

Comment:
Whoops, japicmp has flagged this!

Just as we did in `GenericContainer`, I'll reinstate the previous method signature for binary compatibility.

Reply:
Reinstated.